models/index.js

```diff
@@ -1,3 +1,11 @@
-const User = require('./User');
+const express = require('express');
```

Comment:
I don't think this is where all of these variables belong. Most of them are in the server.js file in the root of the project. The index file in the models folder is used for defining the relations between the models that we're using in the project (how customers relate to clients and vice versa)